spec/dor/governable_spec.rb

```diff
@@ -11,6 +11,12 @@ class GovernableItem < ActiveFedora::Base
   before(:each) { stub_config   }
   after(:each)  { unstub_config }
 
+  before :each do
+    @item = instantiate_fixture('druid:oo201oo0001', Dor::AdminPolicyObject)
+    # @item.stub(:new_record? => false)
+    allow(Dor::Collection).to receive(:find).with('druid:oo201oo0002').and_return(mock_collection)
+  end
+
   let(:mock_collection) {
```

Comment:
No substantial changes to governable_spec, just moved some methods around and added some specs for 100% coverage.  Moved some specs to permissable_spec.